### PR TITLE
Hotfix for issue 3277

### PR DESCRIPTION
**Pull request by Lemongrass3110**
* **Addressed Issue(s)**: #3277

* **Server Mode**: Both

* **Description of Pull Request**: 
Fixes losing state of disable_items in certain scenarios.

### CODE REVIEW

File: src/map/clif.cpp [hunk 1]
```diff
@@ -12160,10 +12160,9 @@ void clif_parse_UseSkillToId(int fd, struct map_session_data *sd)
 			clif_msg(sd, WORK_IN_PROGRESS);
 			return;
 		}
-#else
+#endif
 		if (!sd->npc_item_flag || !(inf&INF_SELF_SKILL))
 			return;
```
Comment (cydh) [inline src/map/clif.cpp]: thow this with clif wip message

File: src/map/status.cpp [hunk 2]
```diff
@@ -4227,6 +4228,10 @@ int status_calc_pc_(struct map_session_data* sd, enum e_status_calc_opt opt)
 	if( (skill = pc_checkskill(sd,SU_SPRITEMABLE)) > 0 && !sd->sc.data[SC_SPRITEMABLE] )
 		sc_start(&sd->bl, &sd->bl, SC_SPRITEMABLE, 100, 1, -1);
 
+	if( previous_st ){
+		script_attach_state(previous_st);
```
Comment (ecdarreola) [inline src/map/status.cpp]: Causes error in latest commit.

Comment (Lemongrass3110) [reply src/map/status.cpp]: Why would it? I think you merged badly.

Comment (ecdarreola) [reply src/map/status.cpp]: My bad it was causing the other error. I'l post it in conversation. Sorry.

**Pull request merged**